tests/globalhelper/operator.go

```diff
@@ -0,0 +1,79 @@
+package globalhelper
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	olmv1 "github.com/operator-framework/api/pkg/operators/v1"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+)
+
+func DeployOperator(namespace string, operatorGroup *olmv1.OperatorGroup, subscription *v1alpha1.Subscription) error {
```

Comment:
It's going to be used only in affiliatedcertification suite. Please move these func under affiliatedcertification folder